modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -74,6 +74,10 @@ object method {
 
     val l0SeedlistPath = seedlistPath
 
+    val prioritySeedlistPath = None
+
+    val realignWithNonPrioritySeedlist = false
```

Comment:
should it be configurable per node or per network?

Reply:
☝️ @ryle-goehausen

Reply:
I would consider a strategy where we always check the majority, but in case realign peers exist we limit the check to those peers only.

Reply:
Per node I believe, if you mean should this be exposed on the CLI of each process, which should control the behavior of that then yes. Otherwise a bit confused by what you mean by per network?

Reply:
I guess it means per layer.

Reply:
`per node` - each node can specify own priority peers to select majority from, so two different nodes on mainnet would be able to query different priority peers so also choose a different majority

`per network` - there should be a single priority peers list for mainnet, another one for testnet, etc.. and all the nodes within a single network should query the same peers for majority selection

Reply:
@TheMMaciek is correct. I understand each layer as a separate network, so `per layer = per network` in above message

Reply:
From discussion: we can remove this flag because we are missing the 3rd CLI parameter to pass in a GL0 seed list to distinguish it from the others. Let's rely entirely on the priority seed list parameter / default embedded value. And then remove this flag because we don't have an appropriate fallback since there are Nones for the L0 seed list entries -- there's no distinction between that and the other seed list where we could allow this parameter. So we're missing the pieces to make use of this additional parameter and need to remove it.